pkg/gitlab/datasource.go

```diff
@@ -14,12 +14,23 @@ type Datasource struct {
 	client *gitlab.Client
 }
 
-// HandleCommitsQuery is the query handler for listing GitLab Commits
+// HandleCommitsQuery is the query handler for listing GitHub Commits
 func (d *Datasource) HandleCommitsQuery(ctx context.Context, query *models.CommitsQuery, req backend.DataQuery) (dfutil.Framer, error) {
-	opt := models.CommitsOptionsWithRepo(query.Options, query.ProjectID)
+	opt := models.CommitsOptionsWithRepo(query.Options, query.Owner, query.Repository)
 	return GetCommitsInRange(ctx, d.client, opt, req.TimeRange.From, req.TimeRange.To)
 }
 
+// HandleLabelsQuery is the query handler for listing GitHub Labels
+func (d *Datasource) HandleLabelsQuery(ctx context.Context, query *models.LabelsQuery, req backend.DataQuery) (dfutil.Framer, error) {
```

Comment:
```suggestion
func (d *Datasource) HandleLabelsQuery(ctx context.Context, query *models.LabelsQuery, _ backend.DataQuery) (dfutil.Framer, error) {
```
I'd ignore the argument if not used.